docs/user/openstack/known-issues.md

```diff
@@ -32,3 +31,32 @@ Some OpenStack clouds do not set default DNS servers for the newly created subne
 If you are in this situation, you can add resolvers to the provisioned subnet by setting the [`externalDNS` property in `install-config.yaml`](./customization.md#cluster-scoped-properties).
 
 Alternatively, for UPI, you will need to [set the subnet DNS resolvers](./install_upi.md#subnet-dns-optional).
+
+## Cluster destruction if Terraform metadata has been lost
```

Comment:
metadata.json has nothing to do with terraform.

Reply:
Ack, I'll fix this